WcaOnRails/app/views/static_pages/organizations.html.erb

```diff
@@ -6,36 +6,24 @@
   <p><%= t('regional_organizations.content') %></p>
 
   <% organizations_info = [
-       {
-         country: "Africa",
-         name: "African Cubing Association",
-         url: "http://www.africancubing.org",
-         logo: "aca.jpg",
-       },
        {
          country: "Australia",
          name: "Speedcubing Australia",
          url: "http://www.speedcubing.org.au",
          logo: "Cube_FA_RGB_pos.png",
        },
        {
-         country: "Brazil",
-         name: "Associação Brasileira de Cubo Mágico",
-         url: "http://abcubomagico.com.br",
-         logo: "abcm.png",
+         country: "Belarus",
+         name: "Speedcubing Federation",
+         url: "http://www.scfed.by",
```

Comment:
Definitely not a blocker for this PR, but do you think we could start requiring regional organizations to run their websites on HTTPS instead of HTTP?

Reply:
Actually, we don't even require a website yet (see https://github.com/thewca/wca-documents/issues/44). When we do, I'm fine with requiring HTTPS!

Reply:
What is the reason to require organizations to run their websites on HTTPS?

Reply:
@cubewhiz, let's continue this discussion over on https://github.com/thewca/wca-documents/issues/44.